src/utils/constants.ts

```diff
@@ -0,0 +1,27 @@
+// SCENE 2 CONSTANTS
+export const CLOUD_WIDTH_MIN = 100;
+export const CLOUD_WIDTH_MAX = 120;
+export const CLOUD_HEIGHT_MIN = 60;
+export const CLOUD_HEIGHT_MAX = 80;
+
+export const NUM_CLOUD_SECTION_PARTICLES = 150;
+export const PARTICLE_DIAMETER = 15;
+
+export const CLOUD_OFFSETS = [
+  {x: -1 * window.innerWidth / 3, y: -1 * window.innerWidth / 14 },
```

Comment:
Was innerWidth chosen for better calibrated scalability? instead of innerHeight

Reply:
Nope! that's definitely a mistake